code/frameworks/nextjs/src/portable-stories.ts

```diff
@@ -0,0 +1,125 @@
+import {
+  composeStory as originalComposeStory,
+  composeStories as originalComposeStories,
+  setProjectAnnotations as originalSetProjectAnnotations,
+  composeConfigs,
+} from '@storybook/preview-api';
+import type {
+  Args,
+  ProjectAnnotations,
+  StoryAnnotationsOrFn,
+  Store_CSFExports,
+  StoriesWithPartialProps,
+} from '@storybook/types';
+
+import { INTERNAL_DEFAULT_PROJECT_ANNOTATIONS as reactAnnotations } from '../../../renderers/react/src/portable-stories';
```

Comment:
```suggestion
// ! ATTENTION: This needs to be a relative import so it gets prebundled. This is to avoid ESM issues in Nextjs + Jest setups
import { INTERNAL_DEFAULT_PROJECT_ANNOTATIONS as reactAnnotations } from '../../../renderers/react/src/portable-stories';
```